### ISSUE TITLE
Rename "Meta Search" to "External Sources"

### ISSUE THREAD

**Issue by zackkrida** [WordPress/openverse-frontend#1197]
## Problem

We currently use two names "Meta Search" and "Additional Sources" to refer to the same feature: letting users search a list of external search engines that also contain CC-licensed content. I think "Additional Sources" is much more sensible for non-technical users. Additionally, using the word "Meta" in the context of our name "Openverse", as it relates to the "Metaverse" is also quite unfortunate.

## Description

Let's remove all references to 'meta search' and replace them with 'additional sources', including internal naming of code, methods, string keys, and so on.

We need to make sure a redirect is in place to preserve url so that `https://wordpress.org/openverse/meta-search/` redirects to `https://wordpress.org/openverse/additional-sources/`.

## Implementation

- [ ] 🙋 I would be interested in implementing this feature.

**Comment by panchovm** [comment]
The word "additional sources" is confusing term as we are not adding anything to Openverse. 

@sarayourfriend suggested in an internal conversation that “external sources” sounds more explicit about what will happen. And I agree with that point.

This term also relates to the "additional types" designed for the switcher popover to show non-supported-yet content types, like video. In that vein, it sounds reasonable revisiting both.

Based on the current pages and designs we made, here are the places that @obulat identified where we are using one or both terms.

## Content switcher

Ticket #739 proposed creating an "additional types" section for the upcoming content types. The feature is not live yet but I think we are on time to revisit it.

<img src="https://user-images.githubusercontent.com/895819/158441163-f7b2ac22-34a9-444b-b458-08aaf9909d65.gif" width="350">

Replacing "additional types" with "coming soon", as you suggested internally @zackkrida, is very accurate and explicit about what this content type belongs to.

## Meta search section

For this section, the current implementation is outdated and ticket #734 addresses this change. It is currently blocked since it waits for this copy definition. Here is how it looks on desktop.

<img src="https://user-images.githubusercontent.com/895819/151384759-80cc75d9-a131-45f8-b1d0-be6c2d098f81.png" width="1400">

In this case, I would replace the “Try another source” with “Try an external source” to follow the idea shared initially.

## Non-supported content type (video)

In previous conversations, we decided to create a page for the upcoming content types to show the existing external sources. We started with video for now, but this page applies to any future content type we decide to support.

Here is a screenshot of a design I am working on and plan to share once we find a path for this discussion.

![image](https://user-images.githubusercontent.com/895819/181094969-ca8fc34c-4f00-4028-9271-40afa6f0cf58.png)

For this page, I would replace “Try one of the additional sources below” with “Try one of the external sources below.”

## Meta search page

Finally, the [meta search page](https://wordpress.org/openverse/meta-search) needs multiple updates by replacing and adding these two terms ("external sources" and "coming soon"). Something that we can do in parallel to control this ticket's scope.

Let me know what you all think of these changes.

**Comment by panchovm** [comment]
As the comment above lists the changes necessary for moving out from "additonal sources" to "external sources", I am linking the tickets related to this update:

### Content switcher
- [ ] Replacing "additional types" with "coming soon" in #1665 

### Metasearch section
- [ ] Replacing “Try another source” with “Try an external source” in the metasearch section, next to the dropdown menu in #1665 

### Non-supported content type
- [ ] Replacing “Try one of the additional sources below” with “Try one of the external sources below". No issue created yet.

### Metasearch page
- [ ] Replacing and adding the "external sources" and "coming soon" phrases. No issue created yet.

**Comment by panchovm** [comment]
@zackkrida The "Non-supported content type" refers to the non-supported page, like video results. I am not sure if you linked the incorrect issue or if we are tackling that change in #1665.

@dhruv mentioned the change I pointed out in [this comment](https://github.com/WordPress/openverse-frontend/issues/734#issuecomment-1168351861).

**Comment by zackkrida** [comment]
Reopening for the page changes needed in #1766

**Comment by zackkrida** [comment]
@panchovm you are right, I made a new issue to track the design and implementation of those pages.

**Comment by zackkrida** [comment]
@panchovm you are right about the 'not supported' pages, I've made a new issue for them here https://github.com/WordPress/openverse-frontend/issues/1791
